dwave/cloud/solver.py

```diff
@@ -1174,6 +1174,212 @@ def check_problem(self, linear, quadratic):
+        described in the
+        `system documentation <https://docs.dwavesys.com/docs/latest/c_qpu_timing.html>`_.
+
+        Requires that you provide the number of qubits to be used for your
+        problem submission. :term:`Embedding` is typically heuristic and the number
+        of required qubits can vary between executions. If you are using a heuristic
+        embedding tool such as
+        :std:doc:`minorminer <oceandocs:docs_minorminer/source/sdk_index>` indirectly
+        through your sampler (e.g., by using the
+        :class:`~dwave.system.composites.EmbeddingComposite` or :class:`~dwave.system.samplers.DWaveCliqueSampler` classes), you can use
+        the same tool on your problem to estimate the expected number of qubits:
+        for large, complex problems you might run the tool several times and take
+        the number of qubits from the produced average or worst-case embedding;
+        for small, simple problems even a single run might be sufficient. If you
+        are using such a tool directly (e.g., in conjunction with the
+        :class:`~dwave.system.composites.FixedEmbeddingComposite`)
+        or otherwise generating a heuristic or non-heuristic embedding, take
+        the required number of qubits from your embedding. Because embedding
+        depends on a QPU’s working graph, such embeddings should be for
+        the particular QPU for which you are estimating the access time.
+
+        Args:
+            num_qubits:
+                Number of qubits required to represent your binary quadratic model
+                on the selected solver.
+            num_reads:
+                Number of reads. Provide this value if you explictly set ``num_reads``
+                in your submission.
+            annealing_time:
+                Annealing duration. Provide this value of if you set
+                ``annealing_time`` in your submission.
+            anneal_schedule:
+                Anneal schedule. Provide the ``anneal_schedule`` if you set it in
+                your submission.
+            initial_state:
+                Initial state. Provide the ``initial_state`` if your submission
+                uses reverse annealing.
+            reinitialize_state:
+                Set to ``True`` if your submission sets ``reinitialize_state``.
+            programming_thermalization:
+                programming thermalization time. Provide this value if you explictly
+                set a value for ``programming_thermalization`` in your submission.
+            readout_thermalization:
+                Set to ``True`` if your submission sets ``readout_thermalization``.
+            reduce_intersample_correlation:
+                Set to ``True`` if your submission sets ``reduce_intersample_correlation``.
+
+        Returns:
+            Estimated QPU access time, in microseconds.
+
+        Raises:
+            KeyError: If a solver property, or a field in the ``problem_timing_data``
```

Comment:
I don't think `KeyError` is raised anymore.

Reply:
Copy & paste error when I split the ``try:`` statement. Fixed